src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork(){
+
+    }
+
+    @Override
+    public void stopWork(){
+

```

Comment:
not implemented (should be fixed in other classes too):

> In those classes override doWork(), so it will print a message that a certain machine started its work.
> Override stopWork() as well. It should print messages that certain machines stopped working.

```suggestion
        System.out.println("Bulldozer started to work!");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer stopped it's work!");
```